test/schema/artwork/index.js

```diff
@@ -47,18 +43,11 @@ describe("Artwork type", () => {
       sale_ids: ["sale-id-not-auction", "sale-id-auction"],
     }
 
-    gravity = sinon.stub()
-    Artwork.__Rewire__("gravity", gravity)
-
     rootValue = {
       artworkLoader: sinon.stub().withArgs(artwork.id).returns(Promise.resolve(artwork)),
     }
   })
 
-  afterEach(() => {
-    Artwork.__ResetDependency__("gravity")
-  })
-
```

Comment:
This one kinda surprised me, but I think it's just that since the artworkLoader is tested, the global rewrite wasn't necessary